pkg/component/component.go

```diff
@@ -344,42 +363,62 @@ func PushLocal(client *occlient.Client, componentName string, applicationName st
 	targetPath := fmt.Sprintf("%s/src", s2iSrcPath)
 
 	// Copy the files to the pod
-	glog.V(4).Infof("Copying to pod %s", pod.Name)
+
+	s := log.Spinner("Copying files to pod")
```

Comment:
I think we can move the `defer s.End(false)` statement below, right after this line. That way we can remove the `s.End(false)` below.

Reply:
The reason no defer has been added here is because this is a separate spinner and we must close it before opening up the second spinner on line 375. If we added a defer, it would mess up the logging.

Reply:
ok :+1: